testkit/backend.py

```diff
@@ -3,8 +3,25 @@
 Assumes driver and backend has been built.
 Responsible for starting the test backend.
 """
-from common import run_in_driver_repo
+from common import open_proccess_in_driver_repo, is_browser
```

Comment:
```suggestion
from common import (
    open_proccess_in_driver_repo,
    is_browser,
)
```

makes git diffs cleaner ;)